documentation/bolt_installing_modules.md

```diff
@@ -329,9 +332,99 @@ module-install:
 - [bolt-project.yaml options](bolt_project_reference.md#module-install)
 - [bolt-defaults.yaml options](bolt_defaults_reference.md#module-install)
 
+## Skip dependency resolution for a module
+
+Sometimes you might want to skip dependency resolution for a specific module,
+usually because Bolt is unable to resolve its dependencies. This allows you to
+take advantage of Bolt's module dependency resolution while still using modules
+that Bolt cannot resolve dependencies for. You might want to skip dependency
+resolution for a module if:
```

Comment:
```suggestion
Skipping dependency resolution for a module allows you to
take advantage of Bolt's module dependency resolution while still using modules
that Bolt cannot resolve dependencies for.

You might want to skip dependency resolution for a module if:
```
Just streamlining this a bit because it's a little redundant with the list